### DISCUSSION TITLE
How to implement guessing_game.py

### DISCUSSION THREAD

**Discussion by itsDYA** [Rapptz/discord.py#6364] [Q&A]
How could I implement the example guessing_game.py to my bot if I'm using discord.ext? I'm very new to this and I don't really know what I'm doing. [My bot](https://github.com/itsDYA/IllyaBot/blob/master/main.py)

**Comment by bandwiches** [comment]
What's the issue you're having?  At first glance it doesn't seem like anything is wrong.  I noticed you're trying to run a `Flask` server in a separate thread from the get-go, I'm wondering if that may be causing an issue if you can't get the `Client` to run.

**Comment by Volfram1235** [comment]
I know that's too late but anyway. Here I've pinned an example of my guessing game.
[guess_num.txt](https://github.com/Rapptz/discord.py/files/6242782/guess_num.txt)
